src/commands.rs

```diff
@@ -578,6 +578,10 @@ pub fn json_get<M: Manager>(manager: M, ctx: &Context, args: Vec<RedisString>) -
 pub fn json_set<M: Manager>(manager: M, ctx: &Context, args: Vec<RedisString>) -> RedisResult {
     let mut args = args.into_iter().skip(1);
 
+    if args.len() < 3 {
+        return Err(RedisError::WrongArity);
+    }
```

Comment:
Why is this needed? Wouldn't the `?` bellow will cover it?

Reply:
you're right, I thought it will be more standard this way

Reply:
Can do preliminary check on the arg count that it is a multiplication of 3